sdk/evaluator.go

```diff
@@ -111,6 +120,20 @@ func (e evaluator) EvaluateRequestPolicy(ctx context.Context, rondInput core.Inp
 		}
 	}
 
+	if options.Audit.Enabled {
```

Comment:
If the policy is not allowed? Should we trace also this kind of access?

Reply:
yes, it makes sense

Reply:
Also, it is needed the flag? If it's used the noop auditAgent, it should be ok to always write it

Reply:
the flat is not needed as I implemented the noop agent for that. I still wanted to better prevent the code execution but it doesn't make much sense. I've removed the flag